docs/user-guide/ingest-data/for-iot/grpc-sdks/template.md

```diff
@@ -27,6 +27,15 @@ The following example shows how to set the username and password when using the
 Each row item in a table consists of three types of columns: `Tag`, `Timestamp`, and `Field`. For more information, see [Data Model](/user-guide/concepts/data-model.md).
 The types of column values could be `String`, `Float`, `Int`, `Timestamp`, `JSON` etc. For more information, see [Data Types](/reference/sql/data-types.md).
 
+## Set table options
+
+Although the time series table is created automatically when writing data to GreptimeDB via this ingester library,
+you can set [table options](/reference/sql/create.md#table-options) such as retention policy, merge mode, and more.
```

Comment:
We support only some of the options in table-options. The currently supported ones include:
1. `auto_create_table` If it is `false`, it means the user knows the table already exists and doesn't need automatic creation, which can improve write performance.
2. `ttl`,`append_mode`,`merge_mode`, as the table options said